src/components/modules/WYSIWYG.js

```diff
@@ -0,0 +1,15 @@
+import React, { Component } from 'react';
+
+class WYSIWYG extends Component {
```

Comment:
This component name was confusing for me. I was asking myself, why are we rendering an WYSIWYG editor on the public page? Until I realized the purpose of this is to inject HTML from that API that originally comes from the WYSIWYG editor in Joplin. Maybe we call it `HtmlFromAPI` or `HtmlFromAdmin`? idk. just thoughts.